FAQ.md

```diff
@@ -58,17 +60,21 @@ types of programming languages.
 Slint is available under either a [proprietary license](LICENSES/LicenseRef-Slint-commercial.md)
 or [GNU GPLv3](LICENSES/GPL-3.0-only.txt)
 
-### Can I license my code under an open-source license more permissive than GPL?
+### If I link my program with Slint GPLv3, does it mean that I have to GPL my program too?
 
-Yes. While the binary or the package containing Slint needs to be licensed under GPL,
-the code you write can be under any license compatible with the GPL, such as BSD, MIT, or Apache license. 
-For more info, you can check the following GPL FAQs:
+No. You can license your program under any license compatible with the GPLv3 such
+as [https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses).
+Refer to GPL FAQ [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL).
 
-- [https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL](https://www.gnu.org/licenses/gpl-faq.en.html#LinkingWithGPL)
-- [https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL](https://www.gnu.org/licenses/gpl-faq.en.html#IfLibraryIsGPL)
+### My MIT-licensed program links to Slint GPLv3. Can someone fork my program to build and distribute a proprietary program?
 
-In other words, you can still allow others to fork your code for proprietary software.
-The person doing that have the choice to either use Slint according to one of our proprietary options, or remove the dependency.
+Yes, provided the person distributing the proprietary program either links to 
+Slint proprietary license instead of Slint GPLv3 or removes the dependency to Slint.
```

Comment:
```suggestion
Yes, provided the person distributing the proprietary program acquired a 
Slint proprietary license instead of using Slint under GPLv3, or removes the dependency to Slint altogether.
```